Cargo.toml

```diff
@@ -27,6 +27,7 @@ num-integer = "0.1.45"
 num-rational = "0.4.1"
 num-traits = "0.2.15"
 smallvec = "1.10"
+symbolic_expressions = { git = "https://github.com/oflatt/symbolic-expressions", rev = "4c0ea5ca008f972450b2af72387e64d2c1c6a791" }
```

Comment:
The symbolic_expressions library has a weird property when printing strings, so I fixed that in my branch